src/golang/cmd/server/handler/preview.go

```diff
@@ -211,11 +191,21 @@ func (h *PreviewHandler) Perform(ctx context.Context, interfaceArgs interface{})
 	artifactResults := make(map[uuid.UUID]previewArtifactResponse)
 	for _, artf := range workflowDag.Artifacts() {
 		if artf.Computed(ctx) {
-			artifactResp, err := convertToPreviewArtifactResponse(ctx, artf)
+			artifact_metadata, err := artf.GetMetadata(ctx)
 			if err != nil {
 				return errorRespPtr, http.StatusInternalServerError, err
 			}
-			artifactResults[artf.ID()] = *artifactResp
+
+			log.Info(artifact_metadata.SerializationType)
```

Comment:
can we use the Logger.logger object here? It's configured by the Client constructor.

Reply:
oops sorry I should have removed this. It's more of a debug logging.